scss/components/_tabs.scss

```diff
@@ -138,7 +138,7 @@ $tab-content-padding: 1rem !default;
   display: none;
   padding: $padding;
 
-  &[aria-hidden="false"] {
+  &[aria-hidden=""] {
```

Comment:
Well yes. I see. It's not about existence of the attribute but about being empty. I think it would be better to fully remove it and not set it to an empty string.